sdk/purview/azure-analytics-purview-scanning/src/test/java/com/azure/analytics/purview/scanning/PurviewScanningClientTestBase.java

```diff
@@ -35,45 +26,23 @@ protected String getEndpoint() {
         return endpoint;
     }
 
-    <T> T clientSetup(Function<HttpPipeline, T> clientBuilder) {
-        TokenCredential credential = null;
-
-        if (!interceptorManager.isPlaybackMode()) {
-            credential = new DefaultAzureCredentialBuilder().build();
-        }
-
-        HttpClient httpClient;
-
-        // Closest to API goes first, closest to wire goes last.
-        final List<HttpPipelinePolicy> policies = new ArrayList<>();
-        policies.add(new RequestIdPolicy());
-        policies.add(new AddDatePolicy());
-
-        HttpPolicyProviders.addBeforeRetryPolicies(policies);
-        if (credential != null) {
-            policies.add(new BearerTokenAuthenticationPolicy(credential, PurviewScanningClientBuilder.DEFAULT_SCOPES));
+    PurviewScanningClientBuilder builderSetUp() {
```

Comment:
Do we need these Purview changes in this PR?

Reply:
I guess this file pops up due to code merge with main. I did not change purview related files in this pr.